rts/lra/test/pom.xml

```diff
@@ -243,6 +243,7 @@
                   <!-- used by start -->
                   <jbossHome>${env.JBOSS_HOME}</jbossHome>
                   <serverConfig>${lra.coordinator.xml.filename}</serverConfig>
+                  <javaOpts>${server.jvm.args}</javaOpts>
```

Comment:
@tomjenkinson this line is overwriting the line that configures the store location on line 241:

```
                  <java-opts>
                    <java-opt>${server.jvm.object.store.arg}</java-opt>
                  </java-opts>
```

Reply:
Thank you for finding this @mmusgrov

I don't think it is a problem with the rebase I did recently as I see that line in an earlier PR from Mayank: https://github.com/jbosstm/narayana/pull/1935/files but that is great to know and gives an idea of where to look - thanks, I will take a look

Reply:
@mmusgrov I have pushed this: https://github.com/jbosstm/narayana/pull/1987/commits/9f4e68071ccf08349aadb063feba0b8ac77c9eef what do you think?

Reply:
I went a different way to what I mentioned before